packages/jest-circus/src/__tests__/after_all-test.js

```diff
@@ -13,7 +13,9 @@
 import {runTest} from '../__mocks__/test_utils';
 
 test('tests are not marked done until their parent afterAll runs', () => {
-  const {stdout} = runTest(`
+  const {stdout} = runTest(
+    'after_all_test',
```

Comment:
test names across the same test file may be the same, because we don't support concurrent test execution. they need to be globally unique though (could calculate a hash or something, but it's too late here XD)